infra/frontend/app-config/prod.tf

```diff
@@ -15,7 +15,7 @@ module "prod_config" {
   instance_desired_instance_count = 4
   instance_scaling_min_capacity   = 4
   # instance_scaling_max_capacity is 5x the instance_scaling_min_capacity
-  instance_scaling_max_capacity = 20
+  instance_scaling_max_capacity = 80
```

Comment:
I need to do calcuations for how much this would cost. In the experience the load testing lasts about an hour, so I need to do the math for the cost of running ... 80 ... containers for an hour.